Add css & html files

sergiogval opened the pull request:
This pull request is made to checkout that linterns are working properly. I had some struggles to make it work, but if you go to the check pages, the 3 checkouts are passed. I also worked on the readme file to comply with reviewers' suggestions.

icebox827 commented on index.html:
```diff
@@ -0,0 +1,12 @@
+<body>
+    <h1 class="hola">Hello, Microverse!!</h1>
+</body>
+</html>
```
- [ ] **REQUIRED:** Project did not pass Linters check. Please make sure that your project pass all the Linters check
- [ ] **REQUIRED:** CSS file is empty, please create the appropriate class in your CSS file as requested in the project requirements.
- [ ] **REQUIRED:** It will be nice to have a descriptive Pull Request with a title and a summary of what has been done in that PR
- [ ] **REQUIRED:** Please add your LinkedIn and Twitter account in the Author section of your README file

sergiogval replied:
done. Please review.

icebox827 reviewed (changes requested):
Hi Sergio 

Good job so far!
Your project looks good, but there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

![](https://media.giphy.com/media/iZKF7CKpEEva2OZE6a/giphy.gif)

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

sergiogval commented:
Finally solved the lintern issues. Added my links to social media in the author section as requested. I've corrected the HTML & CSS files as well. This request comment has more details about the work done, so I think this time I'll pass the code review.

carlos-ssh commented on index.html:
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```
- Fill the README file with the general information about your project, how to install it locally, how to run it. Add your personal information.
- Add a description about the features that you worked on this branch. (this description, it's at the top of your pull request).

sergiogval replied:
I updated my personal info in the readme file, the one that displays when you open the repository. Are you talking about another file?
I also did a description of what I made for this second review. I can't imagine what should I write on the top comment since that request was a bag of issues. Should I describe that?
Please confirm about both topics.

carlos-ssh replied:
You need to fill that template (README file). that you have on this branch with your information. Your template stills with the default information. Kindly fill it with your info project and yours.

sergiogval replied:
I think you are wrong. Please double-check on your statement. The readme file, yes, has the template from microverse. But, no is not the same one. You are killing me. I will escalate this.

sergiogval replied:
![image](https://user-images.githubusercontent.com/67393791/129300413-c56a7516-c540-46ef-b431-9dbdd5583d16.png)

carlos-ssh replied:
This information exists in your branch that you are trying to merge?

krys2fa replied:
Hello @sergiogval 👋,

_This is an additional review as a result of the second opinion request._
@carlos-ssh is not wrong in asking for a detailed README in this branch.
You should've clarified that the README in the master branch is set up correctly with detailed information and that the feature branch is outdated for this file in particular.
This small mistake isn't enough to block you, and we'll fix it for you to get approved.
However, moving forward, you need to make changes on the working branch (not the main one) because CRs only check the branch from the PR. In cases like this, you can update the feature branch so it matches the main one.
The dev team will be contacted to fix the issue.


Keep rocking 🚀!

carlos-ssh reviewed (changes requested):
Hi @sergiogval , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

carlos-ssh reviewed (changes requested):
Hi @sergiogval, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

krys2fa reviewed (commented):
Hello @sergiogval 👋,

This is an additional review as a result of the second opinion request.
@carlos-ssh is not wrong in asking for a detailed README in this branch.
It would've been better to clarify that the README in the master branch is set up correctly with detailed information and that the feature branch is outdated for this file in particular.
This small mistake isn't enough to block you, and we'll fix it for you to get approved.
However, moving forward, you need to make changes on the working branch (not the main one) because CRs only check the branch from the PR. In cases like this, you can update the feature branch so it matches the main one.
The dev team will be contacted to fix the issue.

Keep rocking 🚀!

sergiogval commented:
I'll proceed to merge the feature branch with the master one. Thanks to the CR for their observations, I took note of everything in order to avoid these errors in future projects. I'm happy to finish my very first project on microverse.
